src/Core/Services/SearchService.cs

```diff
@@ -165,5 +175,27 @@ public Task IndexCiphersAsync()
             matchedSends.AddRange(lowPriorityMatchSends);
             return matchedSends;
         }
+
+        private string RemoveDiacritics(string text)
+        {
+            if (string.IsNullOrWhiteSpace(text))
+            {
+                return null;
+            }
+
+            string formD = text.Normalize(NormalizationForm.FormD);
+            StringBuilder sb = new StringBuilder();
+
+            foreach (char ch in formD)
+            {
+                UnicodeCategory uc = CharUnicodeInfo.GetUnicodeCategory(ch);
+                if (uc != UnicodeCategory.NonSpacingMark)
+                {
+                    sb.Append(ch);
+                }
+            }
+
+            return sb.ToString().Normalize(NormalizationForm.FormC);
+        }
```

Comment:
IMO this should be an extension method of `String`. It's easier to find it afterwards and to reuse if needed in other places.